### PR TITLE
Add default split config

### PR DESCRIPTION
**Pull request by hungcs**
Currently on the predibase side, we rely on `defaults` to render preprocessing forms (I assume this will be replaced later by rjsf, but for now, the split form is missing)

we can either
1) Update the defaults here (though not sure this should be the default)
2) Update the split defaults on the predibase engine side
2) Hardcode the split probabilities form on the predibase client side

edit:
```
E               AssertionError: Defaults specified for `preprocessing` but `preprocessing` is
E                               not a feature type recognised by Ludwig.
```
does this mean there is no longer a global preprocessing?

### CODE REVIEW

**Review by ksbrar — APPROVED** [review]
@hungcs I believe that is the correct default!

**Review by arnavgarg1 — APPROVED** [review]
LGTM!

**Comment by arnavgarg1** [comment]
@hungcs

```
E               AssertionError: Defaults specified for `preprocessing` but `preprocessing` is
E                               not a feature type recognised by Ludwig.
```

This error comes from https://github.com/ludwig-ai/ludwig/blob/master/ludwig/utils/defaults.py#L128, where basically it makes sure that if `defaults` is passed into the Ludwig config, that it conforms to the following structure:

```
defaults:
    text:
        preprocessing:
            parameter_1: value
            ...
        encoder:
            parameter_1: value
            ...
        decoder:
            parameter_1: value
            ...
        loss:
            parameter_1: value
            ...
    category:
        preprocessing:
            parameter_1: value
            ...
        encoder:
            parameter_1: value
            ...
        decoder:
            parameter_1: value
            ...
        loss:
            parameter_1: value
            ...
    ...
```

So I guess the Ludwig config wasn't being built correctly, because you might have passed in something along the lines of:

```
defaults:
   preprocessing:
      ....
```

which isn't the correct structure.

Hope this helps!

File: ludwig/utils/defaults.py [hunk 1]
```diff
@@ -63,8 +63,9 @@
 
 default_random_seed = 42
 
+base_preprocessing_split = {"type": "random", "probabilities": list(DEFAULT_PROBABILITIES)}
```
Comment (justinxzhao) [inline ludwig/utils/defaults.py:66]: Let's make this all caps, since it's a constant.

**Review by justinxzhao — APPROVED** [review]

**Pull request merged**